src/app/layout.tsx

```diff
@@ -27,16 +28,17 @@ export default function RootLayout({ children }: LayoutProps) {
     <html lang='en'>
       <body>
         <Providers>
+          <Banner />
           <Header />
           <Flex direction='column' mb='50px'>
             <Flex
               maxWidth='1024px'
               m={['0 auto']}
               p={[
-                '100px 16px 0px 16px',
-                '100px 16px 0px 16px',
-                '128px 16px 0px 16px',
-                '128px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
               ]}
```

Comment:
Could collapse this all into a single `p='48px 16px 0px 16px'` now

Reply:
Fixed in https://github.com/IndexCoop/index-app/pull/991/commits/43e89d42ede8743e18f374aae04efd22778e7df2